### PR TITLE
Enable selenium on travis

### LINKED ISSUE #1251: Discussion - Enable selenium in Travis tests
Looking for feedback on whether or not we should enable selenium tests in Travis. I think there's added value in having them run with the latest versions of python and django. From what I can tell they have never been enabled to run on Travis.

### PR DESCRIPTION
**Pull request by tim-schilling**
Closes #1251

### CODE REVIEW

**Comment by tim-schilling** [comment]
Hmm, this has broken travis.

**Comment by tim-schilling** [comment]
There we go. It's not as clean as I expected, I didn't realize we also had to install `geckodriver`, but it appears to be running those tests successfully.

File: .travis.yml [hunk 1]
```diff
@@ -42,8 +42,13 @@ matrix:
       addons:
         postgresql: "9.5"
     - python: 3.8
-      env: TOXENV=py38-dj31-postgresql
+      env: TOXENV=py38-dj31-postgresql DJANGO_SELENIUM_TESTS=True
+      before_install:
+        - wget https://github.com/mozilla/geckodriver/releases/download/v0.26.0/geckodriver-v0.26.0-linux64.tar.gz
```
Comment (jdufresne) [inline .travis.yml:47]: Is there a reason to avoid 0.27?

**Comment by tim-schilling** [comment]
Nope, I missed that.

**Review by jdufresne — APPROVED** [review]
Otherwise, LGTM. Thanks!

**Comment by tim-schilling** [comment]
Updated to reference v0.270 of geckodriver.

**Comment by matthiask** [comment]
Great, thanks!

**Pull request merged**